packages/text/CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Contributing
+
+Open a PR at https://bitbucket.org/hixme/hixme-ui.
```

Comment:
Maybe change to reference the github repo

Reply:
done!